next.config.js

```diff
@@ -18,6 +18,10 @@ const nextConfig = {
         source: '/reviewed-preprints/:path((?!\\d+v{0,1}\\d*))',
         destination: '/:path*',
       },
+      {
+        source: '/reviewed-preprints/:msid(\\d+v{0,1}\\d*).:format(bib|ris)',
```

Comment:
Should we remove the `ris` until we have implemented that on server?

Reply:
I can remove it and add a reminder of some sort for when we add .ris support?